### DISCUSSION TITLE
Status code Feature

### DISCUSSION THREAD

**Discussion by prabhathulasi** [serverpod/serverpod#1327] [Q&A] [answered]
Is there any status code feature available in the server pod apis. and why there is no official release after v1.1.1 are you stopped the development or its under development

**Comment by odrya12** [comment]
1. You can return from the endpoint a protocol of your own with status code and data.
2. From the roadmap it looks like they working on 1.2 .

**Comment by vlidholt** [comment, marked as answer]
Yes, we are very hard at work on version 1.2 which will be a very major update to Serverpod. Most notable will be features around our tooling (e.g. VS Code plugin) and database (migrations, relations, joins). But there are many many other fixes and features coming too. :)
